samples/csharp_dotnetcore/46.teams-auth/appsettings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "MicrosoftAppId": "",
-  "MicrosoftAppPassword": "",
-  "ConnectionName": ""
+  "MicrosoftAppId": "db2ff7a2-6f8e-4395-8197-46c49f716ce9",
+  "MicrosoftAppPassword": "@1ZCoLfSuoB4h2sU4kSq].[o8-C@/2=U",
```

Comment:
Oops, these should be cleared out.  (I've accidentally done this a number of times too ... this bot's app registration password should also be reset, since it has now become public: https://ms.portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade )

Reply:
Ugh. I committed the wrong file. I meant to commit the maninfest.json file instead. Thanks for catching this.